### ISSUE TITLE
My experimental accuracy is lower than that of the paper

### ISSUE THREAD

**Issue by holayyh** [tongxiao2002/DualGeoSolver#1]
My experiment:
acc: 0.6352604166666667
loss: 1.1794924835364025

My config.json:
{
    "dataset_reader": {
        "type": "s2s_manual_reader",
        "source_token_indexer": {
            "tokens": {
                "type": "pretrained_transformer",
                "do_lowercase": false,
                "model_name": "./chinese-roberta-wwm-ext"
            }
        },
        "target_token_indexer": {
            "tokens": {
                "type": "single_id"
            }
        },
        "tokenizer": {
            "word_splitter": {
                "type": "just_spaces"
            }
        }
    },
    "iterator": {
        "type": "basic",
        "batch_size": 64
    },
    "model": {
        "type": "geo_s2s",
        "beam_size": 10,
        "encoder": {
            "dropout": 0.5,
            "emb_dim": 768,
            "hid_dim": 512,
            "input_dim": 21128
        },
        "knowledge_explanation_file": "./GeoQA-Data/GeoQA-Pro/geometry-knowledges-chinese.json",
        "knowledge_points_ratio": 0,
        "max_decoding_steps": 20,
        "scheduled_sampling_ratio": 0,
        "source_embedder": {
            "token_embedders": {}
        },
        "target_embedding_dim": 512
    },
    "train_data_path": "./GeoQA-Data/GeoQA-Pro/pro_train.pk",
    "validation_data_path": "./GeoQA-Data/GeoQA-Pro/pro_dev.pk",
    "test_data_path": "./GeoQA-Data/GeoQA-Pro/pro_test.pk",
    "trainer": {
        "cuda_device": 0,
        "grad_norm": 10,
        "learning_rate_scheduler": {
            "type": "slanted_triangular",
            "cut_frac": 0.1,
            "num_epochs": 100,
            "num_steps_per_epoch": 108
        },
        "num_epochs": 100,
        "num_serialized_models_to_keep": 2,
        "optimizer": {
            "type": "adam",
            "lr": 0.001,
            "parameter_groups": [
                [
                    [
                        "mcan",
                        "merge_att",
                        "channel_transform",
                        "attflat_img",
                        "attflat_lang",
                        "decode_transform",
                        "inference_transform",
                        "goal_generation_module"
                    ],
                    {
                        "lr": 1e-05
                    }
                ],
                [
                    [
                        "resnet"
                    ],
                    {
                        "lr": 1e-05
                    }
                ],
                [
                    [
                        "source_embedder",
                        "encoder.embedding"
                    ],
                    {
                        "lr": 2e-05
                    }
                ],
                [
                    [
                        "encoder.concat_trans",
                        "encoder.concat_trans_",
                        "encoder.lstm_embedding",
                        "encoder.trans",
                        "encoder.norm",
                        "encoder.concat_norm"
                    ],
                    {
                        "lr": 0.001
                    }
                ]
            ]
        },
        "validation_metric": "+acc"
    },
    "evaluate_on_test": true
}

**Comment by tongxiao2002** [comment]
Maybe it's caused by the physical environments? To mitigate these issues, we further release the checkpoints in the [data link](https://rec.ustc.edu.cn/share/fd0812c0-1dc5-11ef-9485-a15417b8b58d) (under "checkpoints" folder). Hope this would help you.
